datamonster_api/lib/datasource.py

```diff
@@ -1,4 +1,13 @@
+import six
+
 from .base import BaseClass
+from .company import Company
+from .errors import DataMonsterError
+
+try:                # Py2
```

Comment:
❤️  😍 😍 😻

Reply:
Are we really happy about supporting python 2 in the only code we have that we can do py3 😉 ?

Reply:
If we weren't, would we be using six? :)

Reply:
Rhetorical question, mine ;) No, we're not thrilled about it. Py3 is a better language than Py-six.

Reply:
don't we need to support it for both python2 and 3? given that users can call the library from python2 interpreter? i am not familiar enough with its use cases

Reply:
Yes, we do. However, I wonder just how many DM-API users are using Py2? Can't be many, and probably fewer still in < 6 months (1/1/2020).